.gitignore

```diff
@@ -39,6 +39,7 @@ _testmain.go
 
 # Editors
 .idea/
+.vscode/
```

Comment:
This is to avoid git from bothering me about VSCode settings files, which I use for development. This is the similar to the `.idea/` line above for ignoring intellij settings files.